website/static/js/fangorn.js

```diff
@@ -421,11 +511,11 @@ function _fangornDropzoneError(treebeard, file, message) {
         }
         if (child.data.tmpID === file.tmpID) {
             item = child;
-            treebeard.deleteNode(parent.id, item.id);
+            tb.deleteNode(parent.id, item.id);
```

Comment:
`tb` here is undefined.

Again, please use jshint to catch these syntax and style errors.